addons/sale_product_configurator/static/src/js/product_template_attribute_line/product_template_attribute_line.scss

```
@@ -66,27 +47,11 @@
     }
 }
 
-.o_sale_product_configurator_ptav_pills {
-    padding: $spacer/2 $spacer;
-    margin-right: 0.2rem;
-    border: none;
-    cursor: default !important;
-
-    &.btn.active {
-        color: $white;
-        background-color: map-get($theme-colors, 'primary');
-    }
-    &:not(.active) {
-        color: map-get($grays, '600');
-        background-color: map-get($grays, '200');
-    }
-
-    input {
-        -moz-appearance: none;
-        -webkit-appearance: none;
-        appearance: none;
-        opacity: 0;
-    }
+.o_sale_product_configurator_ptav_pills.active label{
+    $-btn-secondary-design: map-get($o-btns-bs-override, "secondary");
```

Missing new line after variables definition:
```suggestion
    $-btn-secondary-design: map-get($o-btns-bs-override, "secondary");
    
```